tests/Makefile.am

```diff
@@ -19,42 +19,6 @@ include $(top_srcdir)/glib-tap.mk
 installed_testdir = $(libexecdir)/installed-tests/eos-updater-@EU_API_VERSION@
 installed_test_metadir = $(datadir)/installed-tests/eos-updater-@EU_API_VERSION@
 
-# Test utility library
-testlib = libeos-updater-test.la
```

Comment:
The reference to `$(testlib)` below needs to be updated to point at the library’s new location.